docs/resources/rms_policy_assignment.md

```diff
@@ -0,0 +1,188 @@
+    resource_type     = "cloudservers"
+    resource_id       = var.ecs_instance_id
+  }
+
+  parameters = {
+    "listOfAllowedFlavors" = "[\"${var.compliant_flavor}\"]"
+  }
+}
+```
+
+### Assign a built-in policy to periodically check whether an OBS bucket is tracked by CTS
+
+```hcl
+variable "policy_assignment_name" {}
+variable "bucket_name" {}
+
+data "huaweicloud_rms_policy_definitions" "test" {
+  name = "cts-obs-bucket-track"
+}
+
+resource "huaweicloud_rms_policy_assignment" "test" {
+  type                 = "builtin"
+  name                 = var.policy_assignment_name
+  description          = "An account is noncompliant if none of its CTS trackers track specified OBS buckets."
+  period               = "Six_Hours"
+  policy_definition_id = try(data.huaweicloud_rms_policy_definitions.test.definitions[0].id, "")
+  status               = "Enabled"
+
+  parameters = {
+    "trackBucket" = "\"${var.bucket_name}\""
+  }
+}
+```
+
+### Assign a custom policy
+
+```hcl
+variable "policy_assignment_name" {}
+variable "function_urn" {}
+variable "function_version" {}
+variable "rms_admin_trust_agency" {}
+
+resource "huaweicloud_rms_policy_assignment" "test" {
+  type        = "custom"
+  name        = var.policy_assignment_name
+  description = "The ECS instances that do not conform to the custom function logic are considered non-compliant."
+  status      = "Enabled"
+
+  custom_policy {
+    function_urn = "${var.function_urn}:${var.function_version}"
+    auth_type    = "agency"
+    auth_value   = {
+      "agency_name": "\"${var.rms_admin_trust_agency}\""
+    }
+  }
+
+  parameters = {
+    "string_example" = "\"string_value\""
+    "array_example"  = "[\"array_element\"]"
+    "object_example" = "{\"terraform_version\":\"1.xx.x\"}"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region where the policy assignment is located.  
+  If omitted, the provider-level region will be used. Changing this will create a new resource.
+
+* `name` - (Required, String) Specifies the name of the policy assignment.  
+  The valid length is limited from can contain `1` to `64`, only letters, digits, hyphens (-) and underscores (_) are
+  allowed. The name must start with a letter.
+
+* `description` - (Optional, String) Specifies the description of the policy assignment, which contain maximum of
+  512 characters.
+
+* `type` - (Optional, String) Specifies the type of the policy assignment.  
+  The valid values are as follows:
+  + **builtin**
+  + **custom**
+
+* `policy_definition_id` - (Optional, String) Specifies the ID of the built-in policy definition.  
+  This parameter and `custom_policy` are alternative.
+
+* `period` - (Optional, String) Specifies the period of the policy assignment.  
+  The valid values are as follows:
+  + **One_Hour**
+  + **Three_Hours**
+  + **Six_Hours**
+  + **Twelve_Hours**
+  + **TwentyFour_Hours**
```

Comment:
ditto.

Reply:
no default value, if omitted, means that the evaluation is performed on all resources under the account.